src/apollo/queries.js

```diff
@@ -38,3 +38,14 @@ export const GET_RUN_TRACKING_DATA = gql`
     }
   }
 `;
+
+/** query for collapsable run details component */
+export const GET_RUN_TRACKING_DATA_WITH_DIFF = gql`
+  query getRunTrackingData($runs: [ID]!, $showDiff: Boolean) {
+    runTrackingData(runIDs: $runs, showDiff: $showDiff) {
+      datasetName
+      datasetType
+      data
+    }
+  }
+`;
```

Comment:
I don't think you need this. We should just be able to add the `showDiff` argument to the `GET_RUN_TRACKING_DATA` query.